uiuc_transportation_demand/calculations.ipynb

```diff
@@ -0,0 +1,518 @@
+    "    \n",
+    "    return round(Fuel_Amount * (CO2_Dictionary[Fuel]),2)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Summary\n",
+    "\n",
+    "Values are not confirmed yet for the GGE and Prices (looking for the most accurate data)\n",
+    "\n",
+    "Lithium-Ion Battery information is hard to come by, will keep searching (seems to be rated only at miles per gallon of gasoline equivalent\n",
+    "\n",
+    "<br>\n",
+    "\n",
+    "__Uncertainties:__\n",
+    "\n",
+    "1) All values for price per unit fuel seem to be varying from source to source with hydrogen and Lithium especially hard to find\n",
+    "\n",
+    "2) Coding errors/mistakes?\n",
+    "\n",
+    "3) Calculating GGE and \\$/GGE incorrectly maybe?\n",
+    "\n",
+    "4) Electic vehicle emissions\n",
+    "\n",
+    "5) E85 emissions per gallon (approximated)\n",
+    "\n",
+    "<br>\n",
+    "\n",
+    "__Still To Do:__\n",
+    "\n",
+    "1) Electric vehicle emissions\n",
+    "\n",
+    "<br>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Sources\n",
+    "\n",
+    "https://www.eia.gov/environment/emissions/co2_vol_mass.php   $\\rightarrow $ CO2 Factors for Gasoline and Diesel\n",
+    "\n",
+    "https://epact.energy.gov/fuel-conversion-factors    $\\rightarrow $ GGE for Gasoline, Diesel, E85 and Hydrogen\n",
+    "\n",
+    "https://www.epa.gov/sites/production/files/2015-07/documents/emission-factors_2014.pdf $\\rightarrow $ CO2 Factor for 100% Ethanol (used 85% * Ethanol CO2 Factor + 15% Gasoline CO2 Factor as an E85 approximation (could not find one))\n",
+    "\n",
+    "https://epact.energy.gov/fuel-conversion-factors    $\\rightarrow $ GGE for electricity\n",
+    "\n",
+    "https://www.elevateenergy.org/wp/wp-content/uploads/Hourly-Pricing-and-EVs-050714.pdf  $\\rightarrow $ Electricity cost per kWh\n",
+    "\n",
+    "Electricity emissions?\n",
+    "\n",
+    "https://gasprices.aaa.com/state-gas-price-averages/     $\\rightarrow$ Illinois Gasoline, Diesel prices\n",
+    "\n",
+    "https://cafcp.org/content/cost-refill#:~:text=Long%20Answer%3A,cost%20of%20%240.21%20per%20mile.   $\\rightarrow$ Hydrogen cost in California\n",
+    "\n",
+    "<br>\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "One gallon of Gasoline is equivalent to 1.0 gallons of Gasoline.\n",
+      "One gallon of Diesel is equivalent to 1.155 gallons of Gasoline.\n",
+      "One gallon of E85 is equivalent to 0.734 gallons of Gasoline.\n",
+      "One kg of Hydrogen is equivalent to 1.019 gallons of Gasoline.\n",
+      "One kWh of Electricity is equivalent to 0.031 gallons of Gasoline.\n"
+     ]
+    }
+   ],
+   "source": [
+    "print('One gallon of Gasoline is equivalent to {} gallons of Gasoline.'.format(GGE_Dictionary['Gasoline']))\n",
```

Comment:
For the sake of readability, my preference is to use f-strings, which looks like
``print(f"One gallon of Gasoline is equivalent to {GGE_Dictionary['Gasoline']} gallons of Gasoline.")``
They look pretty similar, right? But now you don't have to use the long, ``.format()`` AND the variables you want to print are in the string so it reads more like a sentence.